### DISCUSSION TITLE
Fan Speeds Constantly Fluctuating

### DISCUSSION THREAD

**Discussion by Adaptz** [seerge/g-helper#1392] [Q&A]
Hello, I have a 2020 G14 and have a issue with my fan speeds constantly going back and forth from higher to lower rpms while remaining on the same temperature. I only have this issue when using custom fan curves. Even if I use the default custom fan curve the same thing happens. It's not a new issue, I've had this problem since i started using GHelper about a year ago and I still have it. If anyone could help me out that'd be great I've searched everywhere and can't find a solution to this problem.

**Comment by seerge** [comment]
@Adaptz hello

> [How does G-helper control my fan speeds?](https://github.com/seerge/g-helper#how-does-g-helper-control-my-fan-speeds)
> It doesn't. Your BIOS does (same as in case with Armoury). What G-helper can do - is (optionally) set a custom fan profile to current performance mode consisting of 8 pairs of temperature + fan speed % via same endpoint armoury seem to use.

**Reply by Adaptz** [reply]
I'm not quite sure what that has to do with my fan speeds going up and down by like 600rpm while at the same temperature about every 5 seconds.

**Reply by seerge** [reply]
@Adaptz may be due some hardware wear of your fans, they are not able to sustain specific fan speeds? Try to modify your custom fan curve and set a bit different speeds there :) 

Anyway, i can't help here anyhow, as again you can't control fan from app / windows anyhow :) they are controlled by your bios/hardware on low level.

**Comment by luimp3** [comment]
Hey, I had the exact issue with my G14 2020 and the solution was to uncheck the "Apply custom fan curve" from the Ghelper program. It went from fluctuating from 2800rpm to 3400rpm to an stady 2800rpm. I'll use the custom fan curve only when gaming or when using it as a laptop (I use mine mostly as a desktop).
